src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/DoNotDisableHttpClientCRLCheckTests.cs

```diff
@@ -22,10 +21,8 @@ private async Task VerifyCSharpWithDependenciesAsync(string source, params Diagn
         {
             var csharpTest = new VerifyCS.Test
             {
-                TestState =
-                {
-                    Sources = { source, SystemNetHttpApis.CSharp },
```

Comment:
💭 System.Net.Http is included by default in netstandard2.0. Are you seeing errors if we just let it use default?

Reply:
Yes there were errors about `CurlHandler`, `WinHttpHandler` and a third one that I don't remember.

Reply:
I confirm that there are errors when I don't use the custom ref assemblies. I have even tried to force `ReferenceAssemblies.NetStandard.Netstarnd2.0`